code/src/components/Task.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { todolist } from "reducers/todolist";
+import { taskContainer } from "components/taskContainer.css"
+
+export const Task = ({ itemIndex }) => {
+
+    const item = useSelector((store) => store.todolist.list.items[itemIndex]);
+    const dispatch = useDispatch()
+    
+    const onRemoveClicked = event => {
+        dispatch(
+            todolist.actions.remove({
+                itemIndex: itemIndex
```

Comment:
when key and value is the same `itemIndex: itemIndex` your can write it just once `itemIndex`